airbyte-integrations/connectors/source-woocommerce/source_woocommerce/woocommerce.yaml

```diff
@@ -0,0 +1,301 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: [ ]
+  schema_loader:
+    type: JsonFileSchemaLoader
+    file_path: "./source_woocommerce/schemas/{{ options['name'] }}.json"
+  requester:
+    class: source-woo.source_woocommerce.request
```

Comment:
I don't think this line does anything. do we need a custom requster?

Reply:
fixed, extra line